nautobot_golden_config/views.py

```diff
@@ -201,21 +201,24 @@ def queryset_to_csv(self):
         """Export queryset of objects as comma-separated value (CSV)."""
 
         def convert_to_str(val):
-            if bool(val) is False:  # pylint: disable=no-else-return
+            if val is None:
+                return "N/A"
+            if bool(val) is False:
                 return "non-compliant"
-            elif bool(val) is True:
+            if bool(val) is True:
                 return "compliant"
-            return "N/A"
+            return None
```

Comment:
I think we can revert this, this is for csv, which an N/A makes more sense.

Reply:
That outter most return should never get hit. I added the return None to avoid a pylint issue.  But moving the return N/A up to `if val is None` was required or `None` was triggering `non-compliant` to be returned, which is not right.
```python
>>> val = None
>>>
>>> if bool(val) is False:
...     print("non-complaint")
... elif bool(val) is True:
...     print("compliant")
...
non-complaint
```